.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+<!-- Please take a look at our Contributing documentation before submitting a Pull Request!
+https://rook.io/docs/rook/latest/Contributing/development-flow/
+
+Thank you for contributing to Rook! -->
+
+**Description of your changes:**
+
+**Which issue is resolved by this Pull Request:**
+Resolves #
+
+**Checklist:**
+
+- [ ] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/latest/Contributing/development-flow/#commit-structure).
+- [ ] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/latest/Contributing/development-flow/#submitting-a-pull-request)
+- [ ] CI tests has been updated, if necessary.
```

Comment:
How are CI tests different than unit tests and integration tests? They have separate bullets below.

Reply:
right, remove the integration tests section as CI tests make more sense with tool.